lib/graphql.schema.js

```diff
@@ -70,12 +72,16 @@ const _getItemsQueryFromEntity = entity => {
       limit: {
         type: GraphQLInt,
         description: `Default is 10`
+      },
+      sort: {
+        type: _makeSortObjectType(entity)
```

Comment:
This type should array of nonnull ...wrap it with new GraphQLList(_nonnull(...))